scipy/integrate/tests/test_bvp.py

```diff
@@ -152,6 +152,76 @@ def big_sol(x, n):
     return x
 
 
+def big_fun_with_parameters(x, y, p):
+    """ Big version of sl_fun, with two parameters.
+
+    The two differential equations represented by sl_fun are broadcast to the number of rows of y, rotating between the
```

Comment:
Sorry to mention that, but it clearly violates 80 lines margin. We still adhere to it, especially when it doesn't hurt readability too much.

Reply:
Not a problem.